react/client.ts

```diff
@@ -11,26 +8,21 @@ export const getAddress = (
   ).then((res) => res.json())
 
 export const updateSession = (
+  countryCode: string,
   zipCode: string,
   geoCoordinates: number[],
   pickup?: Pickup
-) => {
-  const facetsValue = `zip-code=${zipCode};coordinates=${geoCoordinates.join(
-    ','
-  )}${pickup ? `;pickupPoint=${pickup.pickupPoint.id}` : ''}`
-
-  // __RUNTIME__.segmentToken is not reliable for the facets. It might not be updated. For this reason we must try to get the info from our custom cookie first
-  // Encode to base64 because ";" is not allowed in cookies
-  setCookie(SHIPPING_INFO_COOKIE, btoa(facetsValue))
-
-  return fetch('/api/sessions', {
+  // eslint-disable-next-line max-params
+) =>
+  fetch('/api/sessions', {
     method: 'POST',
-    body: `{"public":{"facets":{"value":"${facetsValue}"}}}`,
+    body: `{"public":{"facets":{"value":"zip-code=${zipCode};country=${countryCode};coordinates=${geoCoordinates.join(
+      ','
+    )}${pickup ? `;pickupPoint=${pickup.pickupPoint.id}` : ''}"}}}`,
```

Comment:
I think that the original position of this code was easier to read, with the formatting of this string being resolved when making a variable attribution, instead of inserting it in the middle of the `fetch` function call. Do you agree?

Also, you removed the `setCookie` function call. Is that the expected behavior?